### PR TITLE
Patch for the export feature

### PR DESCRIPTION
**Pull request by btzr-io**
## Changes
- Set default format to `csv`.
- Add filters for exported data.
- Fix issue  #1157.

### CODE REVIEW

**Comment by tzarebczan** [comment]
Gave it a quick test and it looked good! @IGassmann up to you if you want to merge for this release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
   * Notifications when the channel a user subscribes to uploads new content ([#1066](https://github.com/lbryio/lbry-app/pull/1066))
   * Codacy support for Github contributions ([#1059](https://github.com/lbryio/lbry-app/pull/1059))
   * App category for Linux ([#877](https://github.com/lbryio/lbry-app/pull/877))
-  * Add YouTube Sync reward ([1147](https://github.com/lbryio/lbry-app/pull/1147))
+  * Add YouTube Sync reward ([#1147](https://github.com/lbryio/lbry-app/pull/1147))
```
Comment (IGassmann) [inline CHANGELOG.md]: Good initiative!

Comment (tzarebczan) [reply CHANGELOG.md]: That was my bad :p I'll fix up the changelog before release for others.

**Comment by tzarebczan** [comment]
Thanks for this PR @btzr-io , will send you a tip on Discord :)

**Pull request merged**